tasks/main.yml

```diff
@@ -15,10 +15,25 @@
   ansible.builtin.package:
     name: "{{ packages }}"
 
-- name: Install the cyhy-runner package
+# Only the Debian and Kali distros currently know about the
+# --break-system-packages CLI option.
+- name: Install the cyhy-runner package (not Debian or Kali)
   ansible.builtin.pip:
     executable: pip3
     name: https://api.github.com/repos/cisagov/cyhy-runner/tarball/develop
+  when:
+    - not ansible_distribution is in ["Debian", "Kali"]
+
+- name: Install the cyhy-runner package (Debian and Kali)
+  ansible.builtin.pip:
+    executable: pip3
+    # This is necessary on Debian 12, which correctly recognizes that
+    # the local Python is externally managed (i.e. managed via the
+    # system package manager and not by pip).
+    extra_args: --break-system-packages
+    name: https://api.github.com/repos/cisagov/cyhy-runner/tarball/develop
```

Comment:
@jsf9k - Do you want to put this tarball URL in a variable to prevent someone from accidentally updating one URL and not the other (as we have done elsewhere)?

Reply:
I can do better than that.  Please see commit 96c6e26.

Reply:
Bam - that's fancy! 🚀